test/integration/patients/worklist/schedule.js

```diff
@@ -1441,6 +1457,359 @@ context('schedule page', function() {
+      .find('[data-states-filters-region]')
+      .find('[data-check-region]')
+      .first()
+      .click()
+      .then(() => {
+        const storage = JSON.parse(localStorage.getItem(`reduced-schedule_11111_11111-${ STATE_VERSION }`));
+
+        expect(storage.states).to.deep.equal(['33333']);
+      })
+      .wait('@routeActions')
+      .itsUrl()
+      .its('search')
+      .should('contain', 'filter[clinician]=11111')
+      .should('contain', 'filter[state]=33333')
+      .should('not.contain', 'filter[state]=22222');
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-filters-region]')
+      .find('button')
+      .should('contain', '1');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.sidebar__heading')
+      .should('contain', '1');
+
+    cy
+      .get('@filtersSidebar')
+      .find('[data-states-filters-region]')
+      .find('[data-check-region]')
+      .eq(1)
+      .click()
+      .then(() => {
+        const storage = JSON.parse(localStorage.getItem(`reduced-schedule_11111_11111-${ STATE_VERSION }`));
+
+        expect(storage.states).to.deep.equal([]);
+      })
+      .wait('@routeActions')
+      .itsUrl()
+      .its('search')
+      .should('contain', 'filter[clinician]=11111')
+      .should('contain', `filter[state]=${ NIL_UUID }`);
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-filters-region]')
+      .find('button')
+      .should('contain', '1');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.sidebar__heading')
+      .should('contain', '1');
+
+    cy
+      .get('@filtersSidebar')
+      .find('[data-filter-button]')
+      .first()
+      .click();
+
+    cy
+      .get('.picklist__item')
+      .contains('BCBS PPO 100')
+      .click()
+      .wait('@routeActions');
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-filters-region]')
+      .find('button')
+      .should('contain', '2');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.sidebar__heading')
+      .should('contain', '2');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.js-clear-filters')
+      .click()
+      .then(() => {
+        const storage = JSON.parse(localStorage.getItem(`reduced-schedule_11111_11111-${ STATE_VERSION }`));
+
+        expect(storage.states).to.deep.equal(['22222', '33333']);
+      })
+      .wait('@routeActions')
+      .itsUrl()
+      .its('search')
+      .should('contain', 'filter[clinician]=11111')
+      .should('contain', 'filter[state]=22222,33333')
+      .should('not.contain', 'filter[@insurance]');
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-filters-region]')
+      .find('button')
+      .should('not.contain', '2');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.sidebar__heading')
+      .should('not.contain', '2');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.js-close')
+      .click();
+
+    cy
+      .get('@filtersSidebar')
+      .should('not.exist');
+  });
+
```

Comment:
Should we split the reduced schedule tests into it's own context or a different file?

Reply:
yep. let's do it.

Reply:
Okay I just moved the reduced schedule tests into it's own context.